environments/softlayer/public.yml

```diff
@@ -124,7 +124,7 @@ KSPLICE_ACTIVE: yes
 AMQP_HOST: "{{ groups.rabbitmq.0 }}"
 AMQP_NAME: commcarehq
 
-encrypted_root: "{{ encrypted_root_override|default('/opt/data/ecrypt') }}"
+encrypted_root: "{{ encrypted_root_override|default('/opt/data') }}"
```

Comment:
This ties into your issue comment here: https://github.com/dimagi/commcare-cloud/pull/1429#issuecomment-370766306

I was suggesting (though I didn't state it clearly) that we drop `encrypted_root_override`, define `encrypted_root` on each host where it's not `/opt/data`, _not_ mention it in `public.yml`, and then let the default defined in `ansible/group_vars/all.yml` (`/opt/data`) shine through for `db0`.

Reply:
ok, that sounds fair. I wasn't aware of the value in `all.yml`: 26ca09d